samples/v1.2/Elements/ActionSentiment.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Check with Matt about the naming of this file. He's got some magic that associates element samples with schema documentation based on the name. I think ideally we'd want Action.Sentiment.json, but it might have to be Action.OpenUrl.Sentiment.json to make the association work.

Reply:
@matthidinger how should I name this file to make the magic happen?

Reply:
Naming convention should  be <ElementName>.<PropertyName>.json 
For Actions this is a bit tricky/annoying with the way the site is generated today since you have to copy-paste the file 3 times: `Action.OpenUrl.Sentiment.json`, `Action.ShowCard.Sentiment.json`, `Action.Submit.Sentiment.json` 

I'll file an improvement to the site to reduce this in the future and we can use `Action.Sentiment.json`